src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -36,7 +36,6 @@ public class MessageListViewModel : RxConductor<RxScreen>.Collection.AllActive,
         GeneralHeaderViewModel generalHeaderDisplay;
         string lastSortColumn;
         bool lastSortOrderAscending;
-        int workCount;
```

Comment:
This PR isn't just about introducing nameof. It's also about removing unnecessary `NotifyOfPropertyChange` calls. I plan on eliminating as many as I can, but that's enough for this PR.

As for this change, it's not redundant. By making it a property `PropertyChanged.Fody` weaves in the change notification automatically. `PropertyChanged.Fody` also detects any properties using `WorkCount` like `WorkInProgress` and automatically fires a notification for that property too. That's how I can eliminate the manual `NotifyOfPropertyChange(WorkInProgress)` calls.